css/style.css

```diff
@@ -1,50 +1,275 @@
 /*
 Project Name: Blank Template
 Client: Your Client
-Author: Your Name
+Author: Iko Gabay
 Developer @GA in NYC
 */
 
 /******************************************
 /* SETUP
 /*******************************************/
 
+
+
+
+
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
 html {
   box-sizing: border-box;
 }
 
 *,
 *::before,
 *::after {
-  box-sizing: inherit;
+  box-sizing: inherit
 }
 
-/******************************************
-/* BASE STYLES
-/*******************************************/
 
 body {
-  color: #000;
-  font-size: 12px;
+  color: #1a1a1a;
+  font-size: 26px;
   line-height: 1.4;
-  font-family: Helvetica, Arial, sans-serif;
+  font-family: 'Open Sans'
+  background-color: #f0efef;
+}
+
+a {
+  text-decoration: none;
 }
 
+
 /******************************************
 /* LAYOUT
 /*******************************************/
 
 /* Center the container */
 
-.container {
+
+header {
+  height: 120px;
+  background: #131e27;
+  padding-top: 20px;
   width: 100%;
   margin: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 40px;
+  font-family: 'Open Sans', sans-serif;
 }
 
-header {}
+.navigation a {
+  color: #f9e42e;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 24px;
+}
 
-footer {}
+.navigation a:hover {
+  color: #FFFFFF;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 24px;
+}
+.navigation ul {
+  display: flex;
+  justify-content: flex-end;
```

Comment:
the `justify-content: flex-end;` isn't needed here since the container `.navigation` already separated the `ul` and the `h1` by setting `justify-content: space-between;`